app/forms/school_search_form.rb

```diff
@@ -6,10 +6,12 @@ class SchoolSearchForm
   attr_accessor :school_name, :location, :search_distance, :search_distance_unit, :characteristics, :partnership
 
   def find_schools(page)
-    School.where(
-      "lower(name) like ?", "%#{(school_name || '').downcase}%"
-    ).includes(
-      :network, :lead_provider
-    ).page(page)
+    if partnership&.include? "partnered_with_another_provider"
+      School.where(id: Partnership.select(:school_id).map(&:school_id)).page(page)
+    else
+      School.where("lower(name) LIKE ? OR lower(urn) LIKE ?", "%#{(school_name || '').downcase}%", "%#{(school_name || '').downcase}%").includes(
```

Comment:
I think this would be easier to read split over 4 lines

Reply:
Definitely, my bad on that.